lochness/mediaflux/__init__.py

```diff
@@ -184,13 +196,9 @@ def sync_module(Lochness: 'lochness.config',
                                       stdout=DEVNULL, stderr=STDOUT)
                             p.wait()
 
-                            # verify checksum after download completes if
-                            # checksum does not match, data will be downloaded
-                            # again ENH should we verify checksum 5 times?
-                            cmd += ' --csum-check'
-                            p = Popen(cmd, shell=True,
-                                      stdout=DEVNULL, stderr=STDOUT)
-                            p.wait()
```

Comment:
Does this block not work that you had to write the new code? This block wasn't re-downloading data. It was first calculating checksum of downloaded data to decide if re-download would be necessary. It appears that you are doing  the same thing using information from your `diff_path`.

Or is is that your `diff_path` already contains checksum so you do not want to redo it by this block?